src/components/common/LoginCallbackLoader.js

```diff
@@ -1,9 +1,17 @@
 // istanbul ignore file
-import { LoginCallback } from '@okta/okta-react'; 
+import { LoginCallback } from '@okta/okta-react';
 import React from 'react';
+// import { oktaAuth } from '../../utils/oktaConfig';
+import '@okta/okta-signin-widget/dist/css/okta-sign-in.min.css';
 import { ChildLoadingComponent } from './';
 
+// https://github.com/okta/okta-oidc-js/issues/922
+
 const LoginCallbackLoader = props => {
+  console.log('[CALLBACK FROM OKTA]\n', props);
```

Comment:
probably a good idea to clean-up or remove 
some of these dev-logs / comments 
(before they get lost)

Reply:
I agree. I might have commented on the same ones 😬